tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -287,4 +287,23 @@ Feature: antivirus
       | dav-path-version |
       | old              |
       | new              |
-      | spaces           |
\ No newline at end of file
+      | spaces           |
+
+      
+  Scenario Outline: upload a file with virus and get notification in different languages
+    Given user "Alice" has switched the system language to "<language>"
+    And using <dav-path-version> DAV path
```

Comment:
As this is actually to add coverage for issue https://github.com/owncloud/enterprise/issues/5706, to make sure ue don't get any regressions. Let's add an `issue` tag here so it's easier for reviewers to understand what this scenario is about
```suggestion
  @issue-enterprise-5706
  Scenario Outline: upload a file with virus and get notification in different languages
    Given user "Alice" has switched the system language to "<language>"
    And using <dav-path-version> DAV path
```